_rules/aria-state-or-property-has-valid-value-6a7281.md

```diff
@@ -123,26 +124,18 @@ _There are no major accessibility support issues known for this rule._
 `aria-placeholder` property with valid string value
 
 ```html
-<div role="searchbox" aria-placeholder="MM-DD-YYYY">MM-DD-YYYY</div>
+<div role="searchbox" aria-placeholder="MM-DD-YYYY" aria-label="Your birthdate">MM-DD-YYYY</div>
 ```
 
 #### Passed Example 9
 
```

Comment:
Delete the former example 9 because the attribute is not allowed on the element

Reply:
Should we just change the role then?

Reply:
The test case 9 was illustrating the fact that an attribute not allowed with a certain role, doesn't make the rule failed. Changing the role will juste give us a similar example to what we already have in the rule I think.